<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArgumentManagerDlg</class>
 <widget class="QDialog" name="ArgumentManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>参数管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/img/parameters.png</normaloff>:/new/prefix1/img/parameters.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0">
       <item>
        <widget class="QListWidget" name="listWidget">
         <item>
          <property name="text">
           <string>模板</string>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <stylestrategy>PreferAntialias</stylestrategy>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonAddTemplate">
            <property name="text">
             <string>添加模板</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonDeleteTemplete">
            <property name="text">
             <string>删除模板</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonSetTemplate">
            <property name="text">
             <string>设置模板</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
